### PR TITLE
Tracking ID broadcasting

### PR DESCRIPTION
**Pull request by red-avtovo**
removed unused dev options
bound all switches in dev options
rewrote PreferenceReceiver and StethoApplication in Scala
added broadcast command to get TrackingId for MixPanel
removed some unused imports
#### APK
[Download build #9679](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9679/artifact/build/artifact/wire-dev-PR1216-9679.apk)
[Download build #9680](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9680/artifact/build/artifact/wire-dev-PR1216-9680.apk)
[Download build #9694](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9694/artifact/build/artifact/wire-dev-PR1216-9694.apk)
[Download build #9700](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9700/artifact/build/artifact/wire-dev-PR1216-9700.apk)

### CODE REVIEW

**Comment by red-avtovo** [comment]
build again

File: app/src/experimental/AndroidManifest.xml [hunk 1]
```diff
@@ -17,10 +17,27 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">>
```
Comment (deanrobertcook) [inline app/src/experimental/AndroidManifest.xml]: The extra `>` seems dangerous

File: app/src/main/scala/com/waz/zclient/pages/main/profile/preferences/pages/DevSettingsView.scala [hunk 2]
```diff
@@ -1,45 +1,59 @@
...
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/pages/main/profile/preferences/pages/DevSettingsView.scala]: Maybe just check your style settings here

**Comment by someoneEsle** [comment]
build again

**Comment by someoneEsle** [comment]
When I search for the ID in mixpanel I don't see any activity

**Comment by someoneEsle** [comment]
update: it probably just takes forever for mixpanel to update sometimes

**Pull request merged**